conf/cmd.config

```diff
@@ -20,7 +20,7 @@ params {
     config_profile_description  = 'CMD High performance profile'
 
     // Databases
-    db                          = '/fs1/pipelines/gms_16S-dev/assets/databases/emu_database'
+    db                          = '/fs1/pipelines/taco-dev/assets/databases/emu_database'
```

Comment:
Same here about the path.

Reply:
Marking as resolved, as this is now communicated with Lund.